java/org/apache/catalina/ha/session/SessionMessageImpl.java

```diff
@@ -144,7 +144,7 @@ public String getEventTypeString()
             case EVT_GET_ALL_SESSIONS : return "SESSION-GET-ALL";
             case EVT_SESSION_DELTA : return "SESSION-DELTA";
             case EVT_ALL_SESSION_DATA : return "ALL-SESSION-DATA";
-            case EVT_ALL_SESSION_TRANSFERCOMPLETE : return "SESSION-STATE-TRANSFERED";
+            case EVT_ALL_SESSION_TRANSFERCOMPLETE : return "SESSION-STATE-TRANSFERRED";
```

Comment:
If I should keep the typo for now, I will revert this change.

Reply:
This one is fine. It is only used for logging. And the similar issue in DeltaManager should be OK too.